app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalLogoutController.php

```diff
@@ -58,8 +60,14 @@ public function __invoke(Request $request): JsonResponse
 
         auth('contact')->logout();
 
+        if (ArticleWasHelpful::active() && ! session()->has('guest_id')) {
+            $portalGuest = PortalGuest::create();
+            session()->put('guest_id', $portalGuest->getKey());
+        }
+
         return response()->json([
             'success' => true,
+            'guest_id' => session('guest_id'),
```

Comment:
Do you need to pass `guest_id` here? Is it being used on the Frontend for anything?

Reply:
sorry by mistake i forgot to remove. Now removed.